### ISSUE TITLE
Next line brackets do not raise warnings for a single parameter

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#384]
# Bug report

## What's wrong

Code like:

```python
xy = [1
]
```

does not raise a violation. 

## How is that should be

It should raise a violation. Something related to the parameters or brackets. Maybe both.

**Comment by sobolevn** [comment]
That's not actually true.

Example:
```python
some_set = {1
                     }
```

Output:

```
tests/fixtures/noqa.py:0:1: C101 Coding magic comment not found
tests/fixtures/noqa.py:3:14: C812 missing trailing comma
tests/fixtures/noqa.py:4:12: E124 closing bracket does not match visual indentation
tests/fixtures/noqa.py:4:12: Z318 Found extra indentation
```

Example:

```python
some_set = {1
}
```

Output:

```
tests/fixtures/noqa.py:0:1: C101 Coding magic comment not found
tests/fixtures/noqa.py:3:14: C812 missing trailing comma
tests/fixtures/noqa.py:4:1: E124 closing bracket does not match visual indentation
```

This is good enough.
